### PR TITLE
[New Mock] - Color

### LINKED ISSUE #7: [New Mock] - Color
See data source: https://fakerjs.dev/api/color.html

### PR DESCRIPTION
**Pull request by Eroxl**
Closes #7

Generates a random color in RGB, HSL, HSV, or CMYK
And can export to Hex, Decimal or CSS

### CODE REVIEW

File: modules/colors/api/colors-route.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import getRandomColor from '../utils/getRandomColor';
+import ColorErrors from '../consts/ColorErrors';
+
+const defaultColorSpace = "rgb";
+const defaultColorFormat = "hex";
+
+module.exports = function(app : core.Express){
+
+    app.get("/colors/:qty?", (req: Request, res: Response) => {
+        const qty = req.params.qty ? parseInt(req.params.qty) : 1;
```
Comment (ageddesi) [inline modules/colors/api/colors-route.ts:13]: Hi, Thanks for this awesome work. Do you mind just updating the qty getter to use our util function. getQtyFromRequest

Comment (Eroxl) [reply modules/colors/api/colors-route.ts:13]: Yeah sure

**Comment by ageddesi** [comment]
Thank you @Eroxl for the hard work, happy hacktoberfest, please star the repo if you like what we are doing.

**Comment by ageddesi** [comment]
@all-contributors please add @Eroxl for code

**Pull request merged**